<?xml version="1.0"?>
<launch>
	<node name="pure_pursuit_server" pkg="pure_pursuit" type="pure_pursuit_server" output="screen"/>
</launch>
